src/database/migrations/20201122192732_initial_schema.ts

```diff
@@ -9,7 +9,9 @@ export async function up(knex: Knex): Promise<void> {
             table.string('first_name').notNullable();
             table.string('last_name');
             table.string('email').unique().notNullable();
-            table.string('password').notNullable();
+            table.string('password').defaultTo('NULL');
+            table.string('google_id').defaultTo('NULL');
+            table.string('facebook_id').defaultTo('NULL');
```

Comment:
```suggestion
            table.string('password');
            table.string('google_id');
            table.string('facebook_id');
```
will be fine, since by SQL convention, default is `NULL`